### DISCUSSION TITLE
Add Emote-Sets to Channel Emotes

### DISCUSSION THREAD

**Discussion by connorirvin** [night/betterttv#6696] [Ideas]
NorthernLion kind of touched on this just now on his stream, but I think this is honestly a fantastic idea. I'd contribute to the website if I knew how to contribute to it, even.

The idea:
You have "Packs" of emotes, curated by individuals on the website. Similar to how emotes are uploaded and attributed to a user. This would be a logged in user collecting emotes into a group for easy choosing among streamers.
Examples:
-an entire pack containing all of the currently used popular emotes so that a channel can resonate with users without digging or even without knowing. 
-a pack containing the most popular Peepos (  not pepe >:(  ).
Once clicked the pack adds all of the containing emotes to the channel automatically.
It could even automatically update the channel when the pack changes. 

Consider an old man like NL who can't be bothered to stay trendy on a personal level, but would implement for chat and stay trendy in that fashion. This genuinely would allow non-trendy streamers to stay up on the trendy emotes without caring at all LUL

This would require the packs to have a decent security system though, so that the pack cannot be mis-handled by others.

In essence, as @dclstn has mentioned below, this can be considered an expansion of emote-sets.

**Comment by dclstn** [comment]
BetterTTV has this partially with emote-sets, without the functionality of "adds all of the containing emotes to the channel automatically.". You're really asking for a way to add emote's from an emote-set to a channel.

**Reply by connorirvin** [reply]
I did not know about emote-sets, to be honest. Well "didn't know" is kind of incorrect. It's just not what I'm intending here since it's just a separation for personal/channel/mod reasons. Not really what I'm implying.
But from what I see I suppose an expansion of emote-sets is better wording.
 
Sharable, searchable, addable. Add in a "use" counter and a trending/popular screen for sets.
